app/_src/kubernetes-ingress-controller/concepts/deployment.md

```diff
@@ -344,9 +346,39 @@ separating the control and data flow:
 
 Both of those deployments can be scaled independently.
 
-For more hands on experience with Gateway Discovery please see [our guide][gd-guide]
+For more hands on experience with Gateway Discovery please see [our guide][gd-guide].
 
 [admin]: /gateway/latest/admin-api/
 [gd-guide]: /kubernetes-ingress-controller/{{page.kong_version}}/guides/using-gateway-discovery
 
+### {{site.konnect_short_name}} Integration
+
+{:.warning .no-icon}
+> This feature is released as [beta](/gateway/latest/availability-stages/#beta) and should not be deployed in a production environment.
+
+{{site.kic_product_name}} can be integrated with Kong's [{{site.konnect_short_name}}][konnect] platform. It's an
+optional feature that allows configuring a {{site.konnect_short_name}}'s Runtime Group with the same configuration as the one used
+by {{site.kic_product_name}} for configuring local Kong Gateways. It enables using {{site.konnect_short_name}} UI
+to inspect the configuration of your Kong instances in a **read-only** mode, track [Analytics][konnect-analytics],
+and more.
+
+For installation steps, please see the [Kong Ingress Controller for Kubernetes Association][konnect-kic] page.
+
+![KIC {{site.konnect_short_name}} overview](/assets/images/docs/kubernetes-ingress-controller/kic-konnect-diagram.png "KIC {{site.konnect_short_name}} overview")
+
+From the architecture perspective, the integration is similar to the [Gateway Discovery](#gateway-discovery) and
+builds on top of it. The difference is that {{site.kic_product_name}} additionally configures a Runtime Group in {{site.konnect_short_name}}
+using the public [Admin API][admin] of the {{site.konnect_short_name}}'s Runtime Manager. The connection between {{site.kic_product_name}}
+and {{site.konnect_short_name}} is secured using mutual TLS.
+
+{:.important}
+> {{site.kic_product_name}}'s configured Runtime Group in {{site.konnect_short_name}} is **read-only** and is not used for any runtime
+> traffic. {{site.kic_product_name}} still uses the local Kong instances for proxying. In the case of a connection
+> failure between {{site.kic_product_name}} and {{site.konnect_short_name}}, {{site.kic_product_name}} will continue to operate normally
+> not affecting the basic functionality - only the Runtime Group's configuration in {{site.konnect_short_name}} will not be updated.
```

Comment:
Reworded to try and clarify the roles of the different components. I found the original a bit confusing because the configuration shown in the RG is still _functionally_ used by the proxy. It does still reflect what traffic will be routed where.

While it's not _sent_ from Konnect instances, but that's more an implementation detail than something users need to worry about outside the connectivity loss scenario.

```suggestion
> {{site.kic_product_name}}'s Runtime Group in {{site.konnect_short_name}} is **read-only**.
> Although the configuration displayed in {{site.konnect_short_name}} will match the configuration used by proxy instances, it cannot be modified from the GUI.
> You must still modify the associated Kubernetes resources to change proxy configuration.
> In the event of a connection  failure between {{site.kic_product_name}} and {{site.konnect_short_name}},
> {{site.kic_product_name}} will continue to update data plane proxy configuration normally, but will not
> update the Runtime Group's configuration until it can connect to {{site.konnect_short_name}} again.
```